src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import { Grid, GridJustification, makeStyles } from '@material-ui/core';
+import { ITextResourceBindings } from 'src/features/form/layout';
+import { HelpTextContainer } from 'src/features/form/components/HelpTextContainer';
+import { IRuntimeState } from '../../types';
+
+export interface IImageProps {
+  id: string;
+  image: IImage;
+  textResourceBindings: ITextResourceBindings;
+  language: any;
+  getTextResource: (key: string) => string;
+  getTextResourceAsString: (key: string) => string;
+}
+
+export interface IImage {
+  src: IImagesrc;
+  width: string;
+  align: GridJustification;
+}
+
+export interface IImagesrc {
+  nb: string;
+  nn: string;
+  en: string;
+}
+
+const useStyles = makeStyles({
+  spacing: {
+    letterSpacing: '0.3px',
+  },
+});
+
+export function ImageComponent(props: IImageProps) {
+  const classes = useStyles();
+  const language: string =
+    useSelector((state: IRuntimeState) => state.profile.profile.profileSettingPreference.language);
+  const width = props.image.width || '100%';
+  const align = props.image.align || 'center';
+  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg) || 'image';
+
+  let imgSrc;
+  if (language === 'en' && props.image.src.en) {
+    imgSrc = props.image.src.en;
+  } else if (language === 'nn' && props.image.src.nn) {
+    imgSrc = props.image.src.nn;
+  } else {
+    imgSrc = props.image.src.nb;
+  }
+
+  let showImg;
+  const imgType = imgSrc.slice(-3);
+  if (imgType.toLowerCase() === 'svg') {
```

Comment:
this markup can be moved inside the component and use ternary instead of an if statement.

Reply:
Uh.. I don't think I understand... 😅

Reply:
Inside what component?

Reply:
Sorry, I meant inside the markup of the component instead of using a `{showImg}` it could be:
```js
 { imgType.toLowerCase() === 'svg'
      ? (<object
        type='image/svg+xml'
        id={props.id}
        data={imgSrc}
        width={width}
      >
        <img
         src={imgSrc}
          alt={altText}
        />
      </object>)
      : (<img
        id={props.id}
        src={imgSrc}
        alt={altText}
        width={width}
      />)}
```

Reply:
Alternatively you can use the ternary to assign `showImage` directly as a constant instead of using let.

Reply:
Oh! Yeah, I can do that 😊

Reply:
Here!